### ISSUE TITLE
org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT.testParallelRestoreOperationsFromSingleSnapshot fails on 6.8 windows

### ISSUE THREAD

**Issue by atorok** [elastic/elasticsearch#45650]
## Example build failure
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+multijob-windows-compatibility/70/os=windows-2012-r2/testReport/junit/org.elasticsearch.snapshots/SharedClusterSnapshotRestoreIT/testParallelRestoreOperationsFromSingleSnapshot/

## Reproduction line 
does not reproduce locally
```
gradlew :server:integTest -Dtests.seed=33B9BB0DD51D8EF7 -Dtests.class=org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT -Dtests.method="testParallelRestoreOperationsFromSingleSnapshot" -Dtests.security.manager=true -Dtests.locale=ar-YE -Dtests.timezone=Asia/Rangoon -Dcompiler.java=12 -Druntime.java=8
```

## Example relevant log:
```
java.util.concurrent.ExecutionException: RepositoryException[[test-restore-snapshot-repo] could not read repository data from index blob]; nested: AccessDeniedException[C:\Users\jenkins\workspace\elastic+elasticsearch+6.8+multijob-windows-compatibility\os\windows-2012-r2\server\build\testrun\integTest\J1\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_33B9BB0DD51D8EF7-001\tempDir-002\repos\qsebVVLsgb\pending-incompatible-snapshots-F2g4ZFd6Q3Wz2YLfStgbxw -> C:\Users\jenkins\workspace\elastic+elasticsearch+6.8+multijob-windows-compatibility\os\windows-2012-r2\server\build\testrun\integTest\J1\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_33B9BB0DD51D8EF7-001\tempDir-002\repos\qsebVVLsgb\incompatible-snapshots];
	at __randomizedtesting.SeedInfo.seed([33B9BB0DD51D8EF7:F70D98C0928EFE48]:0)
	at org.elasticsearch.common.util.concurrent.BaseFuture$Sync.getValue(BaseFuture.java:265)
	at org.elasticsearch.common.util.concurrent.BaseFuture$Sync.get(BaseFuture.java:252)
	at org.elasticsearch.common.util.concurrent.BaseFuture.get(BaseFuture.java:94)
	at org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT.testParallelRestoreOperationsFromSingleSnapshot(SharedClusterSnapshotRestoreIT.java:3733)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1750)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:938)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:974)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:988)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
	at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
	at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
	at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
	at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:947)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:832)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:883)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:894)
	at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
	at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
	at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
	at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
	at java.lang.Thread.run(Thread.java:748)
Caused by: RepositoryException[[test-restore-snapshot-repo] could not read repository data from index blob]; nested: AccessDeniedException[C:\Users\jenkins\workspace\elastic+elasticsearch+6.8+multijob-windows-compatibility\os\windows-2012-r2\server\build\testrun\integTest\J1\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_33B9BB0DD51D8EF7-001\tempDir-002\repos\qsebVVLsgb\pending-incompatible-snapshots-F2g4ZFd6Q3Wz2YLfStgbxw -> C:\Users\jenkins\workspace\elastic+elasticsearch+6.8+multijob-windows-compatibility\os\windows-2012-r2\server\build\testrun\integTest\J1\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_33B9BB0DD51D8EF7-001\tempDir-002\repos\qsebVVLsgb\incompatible-snapshots];
	at org.elasticsearch.repositories.blobstore.BlobStoreRepository.getRepositoryData(BlobStoreRepository.java:695)
	at org.elasticsearch.snapshots.RestoreService.restoreSnapshot(RestoreService.java:181)
	at org.elasticsearch.action.admin.cluster.snapshots.restore.TransportRestoreSnapshotAction.masterOperation(TransportRestoreSnapshotAction.java:77)
	at org.elasticsearch.action.admin.cluster.snapshots.restore.TransportRestoreSnapshotAction.masterOperation(TransportRestoreSnapshotAction.java:40)
	at org.elasticsearch.action.support.master.TransportMasterNodeAction.masterOperation(TransportMasterNodeAction.java:124)
	at org.elasticsearch.action.support.master.TransportMasterNodeAction$AsyncSingleAction$2.doRun(TransportMasterNodeAction.java:211)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:751)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	... 1 more
Caused by: java.nio.file.AccessDeniedException: C:\Users\jenkins\workspace\elastic+elasticsearch+6.8+multijob-windows-compatibility\os\windows-2012-r2\server\build\testrun\integTest\J1\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_33B9BB0DD51D8EF7-001\tempDir-002\repos\qsebVVLsgb\pending-incompatible-snapshots-F2g4ZFd6Q3Wz2YLfStgbxw -> C:\Users\jenkins\workspace\elastic+elasticsearch+6.8+multijob-windows-compatibility\os\windows-2012-r2\server\build\testrun\integTest\J1\temp\org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT_33B9BB0DD51D8EF7-001\tempDir-002\repos\qsebVVLsgb\incompatible-snapshots
	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:83)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
	at sun.nio.fs.WindowsFileCopy.move(WindowsFileCopy.java:301)
	at sun.nio.fs.WindowsFileSystemProvider.move(WindowsFileSystemProvider.java:287)
	at org.apache.lucene.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:147)
	at org.apache.lucene.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:147)
	at org.apache.lucene.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:147)
	at org.apache.lucene.mockfile.FilterFileSystemProvider.move(FilterFileSystemProvider.java:147)
	at java.nio.file.Files.move(Files.java:1395)
	at org.elasticsearch.common.blobstore.fs.FsBlobContainer.moveBlobAtomic(FsBlobContainer.java:175)
	at org.elasticsearch.common.blobstore.fs.FsBlobContainer.writeBlobAtomic(FsBlobContainer.java:149)
	at org.elasticsearch.repositories.blobstore.BlobStoreRepository.writeAtomic(BlobStoreRepository.java:833)
	at org.elasticsearch.repositories.blobstore.BlobStoreRepository.writeIncompatibleSnapshots(BlobStoreRepository.java:765)
	at org.elasticsearch.repositories.blobstore.BlobStoreRepository.getRepositoryData(BlobStoreRepository.java:687)
	... 10 more

```

## Frequency

[build-stats](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-21d,mode:relative,to:now))&_a=(columns:!(build_url,branch,summary,build.job-name,process.status,process.took,test),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:e58bf320-7efd-11e8-bf69-63c8ef516157,key:test,negate:!f,params:(query:testParallelRestoreOperationsFromSingleSnapshot,type:phrase),type:phrase,value:testParallelRestoreOperationsFromSingleSnapshot),query:(match:(test:(query:testParallelRestoreOperationsFromSingleSnapshot,type:phrase))))),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'class:%20%22org.elasticsearch.snapshots.SharedClusterSnapshotRestoreIT%22'),sort:!(time,desc))) [requires authentication]

**Comment by original-brownbear** [comment]
Strange, I could've sworn I muted this one. This test won't work on Windows pre `7.4` -> muting it in a bit.

**Comment by atorok** [comment]
I just did

**Comment by atorok** [comment]
Looks like it was muted on `7.3` only in relation to #44671

**Comment by original-brownbear** [comment]
Ah thanks @atorok! then we can close here, this will never be fixed on `6.8`.
